Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKTouchController.cs

```diff
@@ -0,0 +1,119 @@
+using Microsoft.MixedReality.Toolkit.XRSDK.Input;
+using UnityEngine;
+using Unity.Profiling;
+using UnityEngine.XR;
+using Unity.XR.Oculus;
+
+namespace Microsoft.MixedReality.Toolkit.XRSDK.Oculus
+{
+    [MixedRealityController(
+        SupportedControllerType.OculusTouch,
+        new[] { Handedness.Left, Handedness.Right },
+        "StandardAssets/Textures/OculusControllersTouch")]
+    public class OculusXRSDKTouchController : GenericXRSDKController
+    {
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        public OculusXRSDKTouchController(TrackingState trackingState, Handedness controllerHandedness,
+            IMixedRealityInputSource inputSource = null, MixedRealityInteractionMapping[] interactions = null)
+            : base(trackingState, controllerHandedness, inputSource, interactions)
+        {
+        }
+
+        /// <inheritdoc />
+        public override MixedRealityInteractionMapping[] DefaultLeftHandedInteractions => new[]
+        {
+            new MixedRealityInteractionMapping(0, "Spatial Pointer", AxisType.SixDof, DeviceInputType.SpatialPointer),
+            new MixedRealityInteractionMapping(1, "Axis1D.PrimaryIndexTrigger", AxisType.SingleAxis, DeviceInputType.Trigger),
+            new MixedRealityInteractionMapping(2, "Axis1D.PrimaryIndexTrigger Touch", AxisType.Digital, DeviceInputType.TriggerTouch),
+            new MixedRealityInteractionMapping(3, "Axis1D.PrimaryIndexTrigger Near Touch", AxisType.Digital, DeviceInputType.TriggerNearTouch),
+            new MixedRealityInteractionMapping(4, "Axis1D.PrimaryIndexTrigger Press", AxisType.Digital, DeviceInputType.TriggerPress),
+            new MixedRealityInteractionMapping(5, "Axis1D.PrimaryHandTrigger Press", AxisType.SingleAxis, DeviceInputType.GripPress),
+            new MixedRealityInteractionMapping(6, "Axis2D.PrimaryThumbstick", AxisType.DualAxis, DeviceInputType.ThumbStick),
+            new MixedRealityInteractionMapping(7, "Button.PrimaryThumbstick Touch", AxisType.Digital, DeviceInputType.ThumbStickTouch),
+            new MixedRealityInteractionMapping(8, "Button.PrimaryThumbstick Near Touch", AxisType.Digital, DeviceInputType.ThumbNearTouch),
+            new MixedRealityInteractionMapping(9, "Button.PrimaryThumbstick Press", AxisType.Digital, DeviceInputType.ThumbStickPress),
+            new MixedRealityInteractionMapping(10, "Button.Three Press", AxisType.Digital, DeviceInputType.PrimaryButtonPress),
+            new MixedRealityInteractionMapping(11, "Button.Four Press", AxisType.Digital, DeviceInputType.SecondaryButtonPress),
+            new MixedRealityInteractionMapping(13, "Button.Three Touch", AxisType.Digital, DeviceInputType.PrimaryButtonTouch),
+            new MixedRealityInteractionMapping(14, "Button.Four Touch", AxisType.Digital, DeviceInputType.SecondaryButtonTouch),
+            new MixedRealityInteractionMapping(12, "Button.Start Press", AxisType.Digital, DeviceInputType.Menu),
```

Comment:
The numerical ordering of these seems to be off (i.e. it goes 10->11->13->14->13->15->16) - is this by design or a copy pasta? (If by design curious about the reasoning there)

Reply:
it was some fresh pasta 🍝